app/templates/components/main-layout.hbs

```diff
@@ -9,7 +9,6 @@
 
     {{#each @sortedGroupedResults as |result|}}
       <h2 class="anchorable-toc rounded-lg padding-small">Deprecations Added in {{result.since}} </h2>
-      {{renderId}}
```

Comment:
I'm guessing this was a leftover from testing something? Couldn't find any occurrence of `renderId` in the entire codebase.